libraries/AP_LQR_Control/keywords.txt

```diff
@@ -0,0 +1 @@
+AP_LQR_Control KEYWORD1
```

Comment:
Any reason for including this?

Reply:
Nothing specific. I just tried to keep it same as the AP_L1_Control library.

Reply:
Ah that's what I thought. We should remove it, it's an old file from Arduino times that isn't used anymore.